modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/resources/META-INF/resources/js/FDSViews.tsx

```diff
@@ -0,0 +1,236 @@
+						'your-request-failed-to-complete'
+					),
+					type: 'danger',
+				});
+			}
+		};
+
+		const validate = () => {
+			if (!fdsViewLabelRef.current?.value) {
+				setLabelValidationError(true);
+
+				return false;
+			}
+
+			return true;
+		};
+
+		return (
+			<>
+				<ClayModal.Header>
+					{Liferay.Language.get('new-dataset-view')}
+				</ClayModal.Header>
+
+				<ClayModal.Body>
+					<ClayForm.Group
+						className={classNames({
+							'has-error': labelValidationError,
+						})}
+					>
+						<label htmlFor={`${namespace}fdsViewLabelInput`}>
+							{Liferay.Language.get('name')}
+
+							<RequiredMark />
+						</label>
+
+						<ClayInput
+							id={`${namespace}fdsViewLabelInput`}
+							onBlur={() => {
+								setLabelValidationError(
+									!fdsViewLabelRef.current?.value
+								);
+							}}
+							ref={fdsViewLabelRef}
+							type="text"
+						/>
+
+						{labelValidationError && (
+							<ClayForm.FeedbackGroup>
+								<ClayForm.FeedbackItem>
+									<ClayForm.FeedbackIndicator symbol="exclamation-full" />
+
+									{Liferay.Language.get(
+										'this-field-is-required'
+									)}
+								</ClayForm.FeedbackItem>
+							</ClayForm.FeedbackGroup>
+						)}
+					</ClayForm.Group>
+
+					<ClayForm.Group>
+						<label htmlFor={`${namespace}fdsViewDesctiptionInput`}>
+							{Liferay.Language.get('description')}
+						</label>
+
+						<ClayInput
+							id={`${namespace}fdsViewDesctiptionInput`}
+							ref={fdsViewDescriptionRef}
+							type="text"
+						/>
+					</ClayForm.Group>
+				</ClayModal.Body>
+
+				<ClayModal.Footer
+					last={
+						<ClayButton.Group spaced>
+							<ClayButton
+								onClick={() => {
+									const success = validate();
+
+									if (success) {
+										addFDSView();
+									}
+								}}
+							>
+								{Liferay.Language.get('save')}
+							</ClayButton>
+
+							<ClayButton
+								displayType="secondary"
+								onClick={() => closeModal()}
+							>
+								{Liferay.Language.get('cancel')}
+							</ClayButton>
+						</ClayButton.Group>
+					}
+				/>
+			</>
+		);
+	};
+
+	const creationMenu = {
+		primaryItems: [
+			{
+				label: Liferay.Language.get('new-dataset-view'),
+				onClick: ({loadData}: {loadData: Function}) => {
+					openModal({
+						contentComponent: ({
+							closeModal,
+						}: {
+							closeModal: Function;
+						}) => (
+							<AddFDSViewModalContent
+								closeModal={closeModal}
+								loadData={loadData}
+							/>
+						),
+					});
+				},
+			},
+		],
+	};
+
+	const views = [
+		{
+			contentRenderer: 'list',
+			name: 'list',
+			schema: {
+				description: 'description',
+				symbol: 'symbol',
+				title: 'label',
+			},
+		},
+	];
+
+	return (
+		<>
+			<FrontendDataSet
+				apiURL={`${fdsEntriesAPIURL}/${fdsEntryId}/fdsViews`}
+				creationMenu={creationMenu}
+				id={`${namespace}FDSViews`}
+				style="fluid"
+				views={views}
+				{...PAGINATION_PROPS}
+			/>
+		</>
```

Comment:
I think this empty fragment is obsolete.

Reply:
I'll also make this change in `FDSEntries.tsx`. 

I could have sworn that this caused an SF error few days ago. Was this a bug in source formatter that we recently fixed? Maybe I was just imagining things :sweat_smile: